packages/docs/src/components/page-configs/getting-started/configuration-guide/page-config.ts

```diff
@@ -27,9 +27,9 @@ app.use(VuesticPlugin, {
 `
 
 const iconsInstall = `
-yarn add material-design-icons-iconfont -D
+yarn add material-design-icons
 // $t('all.code.or')
-npm install material-design-icons-iconfont -D
```

Comment:
[material-design-icons-iconfont](https://www.npmjs.com/package/material-design-icons-iconfont) -is a ~fork (+ fixes) of [Google's Material Design icons](https://github.com/google/material-design-icons) repository
it means that it could include different icons then in recommended fonts:
```html
<link href="https://fonts.googleapis.com/css2?family=Material+Icons"
      rel="stylesheet">
```      
That is why, changed it to [Google's Material Design icons](https://github.com/google/material-design-icons) `material-design-icons`